app/Http/Controllers/API/Schemas/Bioschema/BiochemaController.php

```diff
@@ -0,0 +1,314 @@
+
+        $studySchema = BioSchema::Study();
+        $studySchema['@id'] = $study->uuid;
+        $studySchema['dct:conformsTo'] = $this->conforms();
+        $studySchema->datePublished($study->updated_at);
+        $studySchema->description($study->description);
+        $studySchema->name($study->name);
+        $studySchema->studyDomain('Chemistry');
+        $studySchema->studySubject('Small molecules');
+        $studySchema->dateCreated($study->created_at);
+        $studySchema->keywords($tags);
+        $studySchema->url($study->public_url);
+        $studySchema->about($sampleSchema);
+        $studySchema->license($study->license->url);
+        $studySchema->hasPart($datasets);
+
+        return $studySchema;
+    }
+
+    public function project($project)
+    {
+        /**
+         * Implement Bioschemas' project.
+         *
+         * @param  App\Models\Project  $project
+         * @return object $projectSchema
+         */
+        $authors = [];
+        foreach ($project->authors as &$author) {
+            $authorSchema = Schema::Person();
+            $authorSchema->givenName($author->given_name);
+            $authorSchema->familyName($author->family_name);
+            $authorSchema->identifier($author->orcid_id);
+            $authorSchema->email($author->email_id);
+            $authorSchema->affiliation($author->affiliation);
+            array_push($authors, $authorSchema);
+        }
+        $tags = [];
+        foreach ($project->tags as &$tag) {
+            $tag = $tag->name;
+            array_push($tags, $tag);
+        }
+        $studies = ['studies' => []];
+        foreach ($project->studies as &$study) {
+            $datasets = ['datasets' => []];
+            foreach ($study->datasets as &$dataset) {
+                $datasetSchema = $this->datasetShort($dataset);
+                array_push($datasets['datasets'], $datasetSchema);
+            }
+            $studySchema = $this->studyShort($study);
+            array_push($studies['studies'], $studySchema);
+        }
+
+        $projectSchema = BioSchema::Project();
+        $projectSchema['@id'] = $project->uuid;
+        $projectSchema['dct:conformsTo'] = $this->conforms();
+        $projectSchema['author'] = $authors;
+        $projectSchema->datePublished($project->updated_at);
+        $projectSchema->description($project->description);
+        $projectSchema->name($project->name);
+        $projectSchema->projectDomain('Chemistry');
+        $projectSchema->projectSubject('Small molecules');
+        $projectSchema->dateCreated($project->created_at);
+        $projectSchema->keywords($tags);
+        $projectSchema->url($project->public_url);
+        $projectSchema->license($project->license->url);
+        $projectSchema->hasPart($studies);
+
+        return $projectSchema;
+    }
+
+    public function name(Request $request, $username, $projectName, $studyName = null, $datasetName = null)
```

Comment:
Consider a more descriptive name such as fetchSchema etc. "name" is not descriptive enough. Also, code comments are missing here.